src/main/java/org/rumbledb/compiler/TranslationVisitor.java

```diff
@@ -333,14 +331,16 @@ public Void visitForClause(JsoniqParser.ForClauseContext ctx) {
     @Override
     public Void visitForVar(JsoniqParser.ForVarContext ctx) {
         VariableReferenceExpression var, atVarRef = null;
```

Comment:
Could we fix this to have one variable per declaration.

Reply:
done